ops/generate-subgraph.js

```diff
@@ -155,18 +158,20 @@ function buildTemplates() {
   let results = [];
 
   forEachTemplate((name, mapping, arcVersion) => {
-    results.push(
-      ...combineFragments(
-        [{
-          name,
-          mapping,
-          arcVersion
-        }],
-        true,
-        undefined,
-        undefined
-      )
-    );
+    if (arcVersion !== '0.1.2-rc.5') {
```

Comment:
why this if is needed?

Reply:
Because we don't have an Arc rc.5... (well not a valid release at least)

Reply:
so why add that to the code here?

Reply:
To skip on it, because the loop will try to create for it as well

Reply:
you need to skip it on migration.